cmd/cerbos/repl/internal/repl.go

```diff
@@ -99,6 +104,7 @@ func NewREPL(reader *liner.State, output Output) (*REPL, error) {
 		parser:   parser,
 		output:   output,
 		toRefVal: conditions.StdEnv.CELTypeAdapter().NativeToValue,
+		fs:       os.DirFS("."),
```

Comment:
Instead of limiting to the current working directory, I think we can create the FS on demand in the `completeFile` method using the parent directory of the `prefix`.

Reply:
This turned out to require dropping fs.FS and using actual files for the test. I've removed fs.FS from code I've written more often than I've left it in, a real shame.

Reply:
Yeah, I still have to fallback to the Afero library for anything that requires more than the most basic read-only operations.